src/etos_client/__main__.py

```diff
@@ -70,14 +68,14 @@ def parse_args(args):
     parser.add_argument(
         "-i",
         "--identity",
-        help="Artifact created identity purl to execute test suite on.",
-        **environ_or_required("IDENTITY")
+        help="Artifact created identity purl or ID to execute test suite on.",
+        **environ_or_required("IDENTITY"),
```

Comment:
This should be IDENTIFIER. Or add check for environment variable IDENTIFIER to not break backwards compatibility.

Reply:
I will not do this rename. Let's write an issue to fix this in the future. This is a breaking change and requires deprecation.

Reply:
Agreed!